modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1385,6 +1386,188 @@ Verify that this is the intended server.
 
 ====
 
+[#_schema_notifications]
+== `SCHEMA` category
+
+Schema notifications provide additional information related to indexes and constraints.
+
+[#_neo_clientnotification_schema_indexorconstraintalreadyexists]
+=== IndexOrConstraintAlreadyExists
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Schema.IndexOrConstraintAlreadyExists
+|Title
+a|`<command>` has no effect.
+|Description
+a|`<conflicting>` already exists.
+|Severity
+m|INFORMATION
+|Category
+m|SCHEMA
+|===
+
+.Creating an index when an equivalent index already exists
+====
+Given a range index on `(:Label \{property})` named `existingRangeIndex`.
+
+Command::
++
+[source, cypher]
+----
+CREATE INDEX labelProperyRangeIndex IF NOT EXISTS FOR (n:Label) ON (n.property)
+----
+
+Full title of the returned code::
+`CREATE RANGE INDEX labelProperyRangeIndex IF NOT EXISTS FOR (e:Label) ON (e.property)` has no effect.
+
+Full description of the returned code::
+`RANGE INDEX existingRangeIndex FOR (e:Label) ON (e.property)` already exists.
+
+Suggestions for improvement::
+The existing index is likely what was wanted, no further actions needed.
```

Comment:
I don't like this sentence but I also didn't come up with anything good so some input would be appreciated.